### PR TITLE
Add beginner exercises

### PR DESCRIPTION
**Pull request by ShivaniPR**

### CODE REVIEW

File: SwiftExercises.playground/Sources/Beginner/CandyJar.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import Foundation
+import XCTest
+
+// You are playing a game with a jar full of candies against an opponent.
+// At each step both you and your opponent can take between one and three candies out of the jar.
+// Whoever makes the last move wins. Given numberOfCandies, print true if you are sure you can win and false otherwise.
+
+class CandyJar {
+    let numberOfCandies: Int!
+    var number = 0
```
Comment (chetanfaris) [inline SwiftExercises.playground/Sources/Beginner/CandyJar.swift:10]: Why is this declared global to the class? I see it used only inside the `canWin` function.

File: SwiftExercises.playground/Sources/Beginner/ExpandSequence.swift [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import Foundation
+import XCTest
+
+// You are given an array numbers. Expand the numbers into an a palindrome landscape of mountains.
+// For numbers = [2,3,2,4], palindromeMountains = [1,2,1,2,3,1,2,1,2,3,4,3,2,1] 
+
+class ExpandSequence {
+    let numbers = [2,3,2,4]
+    var k = 0
+    var a:[Int]=[]
```
Comment (chetanfaris) [inline SwiftExercises.playground/Sources/Beginner/ExpandSequence.swift:10]: You need to give meaningful names to the variables. What does this array represent?

File: SwiftExercises.playground/Sources/Beginner/Multiple.swift [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+class Multiple {
+    let numbers = [3, 6, 4, 15, 30]
+    
+    func getSmallestMultiple() -> Int {
+            let num = numbers.sorted()
+            var ans = 0
+            var count = 0
+            var n = num[0]
+            for i in 1...30{
```
Comment (chetanfaris) [inline SwiftExercises.playground/Sources/Beginner/Multiple.swift:14]: Why 30 is the upper limit?

File: SwiftExercises.playground/Sources/Beginner/Multiple.swift [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+                n = n * i
+                count = 0
+                for j in numbers{
+                    if n%j == 0{
+                        count += 1
+                    }
+                    if count == 5{
```
Comment (chetanfaris) [inline SwiftExercises.playground/Sources/Beginner/Multiple.swift:21]: Why `count == 5`?

File: SwiftExercises.playground/Sources/Beginner/PeakCounting.swift [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+// You are given an array of numbers representing the height above the horizon of the mountains from Beautiful British Columbia.
+// Count the number of peaks in the beautiful view. Any number with smaller neighbouring numbers counts as a peak.
+
+class PeakCounting {
+    let numbers = [1, 2, 3, 2, 1, 3, 5, 2, 3, 1]
+    
+    func count() -> Int {
+        var count = 0
+        for i in 1..<9 {
```
Comment (chetanfaris) [inline SwiftExercises.playground/Sources/Beginner/PeakCounting.swift:12]: Why 9?

File: SwiftExercises.playground/Sources/Beginner/PeakFinding.swift [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+// You are given an array of numbers which increase up to a point and then decreases until the end of the array.
+// Find the position where the array stops increasing any further.
+
+class PeakFinding {
+let numbers = [1, 2, 3, 5, 9, 4, 3, 2, 1]
+var ans = 0
+    
+func find() -> Int {
+   var count = 0
+            for i in 0..<9 {
```
Comment (chetanfaris) [inline SwiftExercises.playground/Sources/Beginner/PeakFinding.swift:13]: Why 9?

File: SwiftExercises.playground/Sources/Beginner/CandyJar.swift (hunk 1 above)
Comment (ShivaniPR) [reply SwiftExercises.playground/Sources/Beginner/CandyJar.swift:10]: Yes. It's used only in the canWin function. I'll declare it inside the function.

File: SwiftExercises.playground/Sources/Beginner/ExpandSequence.swift (hunk 2 above)
Comment (ShivaniPR) [reply SwiftExercises.playground/Sources/Beginner/ExpandSequence.swift:10]: The array represents palindromeMountains. I'll change the name of the array.

File: SwiftExercises.playground/Sources/Beginner/PeakCounting.swift (hunk 5 above)
Comment (ShivaniPR) [reply SwiftExercises.playground/Sources/Beginner/PeakCounting.swift:12]: 9 is the length of the array. I'll change it such that it is not defined manually.

File: SwiftExercises.playground/Sources/Beginner/PeakFinding.swift (hunk 6 above)
Comment (ShivaniPR) [reply SwiftExercises.playground/Sources/Beginner/PeakFinding.swift:13]: 9 is the length of the array. I'll change it such that it is not defined manually.

File: SwiftExercises.playground/Sources/Beginner/Multiple.swift (hunk 4 above)
Comment (ShivaniPR) [reply SwiftExercises.playground/Sources/Beginner/Multiple.swift:21]: 5 is the length of the array. I'll change it such that it is not defined manually.

**Pull request merged**